### PR TITLE
paintcode.rb: updated url, app name and changed version to :latest

### PR DESCRIPTION
**Pull request by adelinofaria**

### CODE REVIEW

**Comment by adelinofaria** [comment]
Reference:

https://github.com/caskroom/homebrew-cask/commit/3780e8a6bb598f75392a0242daa42246dc681403

File: Casks/paintcode.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
 cask :v1 => 'paintcode' do
-  version '2.3.2'
-  sha256 '7a91f926ff352659abb5a189d0533c572a2b0bdced93d7b8bb94692d1b9e3487'
+  version :latest
+  sha256 :no_check
 
-  url "http://www.paintcodeapp.com/content/versions/#{version}/paintcode-trial.zip"
+  # pixelcut.com is the official download host per the appcast feed
+  url "http://www.pixelcut.com/paintcode/paintcode.zip"
```
Comment (vitorgalvao) [inline Casks/paintcode.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by adelinofaria** [comment]
Updated @vitorgalvao, thanks for the hint.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by adelinofaria** [comment]
Squashed and updated from caskroom/homebrew-cask

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 4428acedc52b94586f4be2d5c16ddd4a9b801ab7 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**